types/v1/labels.go

```diff
@@ -29,6 +30,7 @@ const (
 // defaults
 type EnvironmentVariableDefaults string
 
+// const is used to define EnvironmentVariableDefaults
```

Comment:
I don't see relevant issue description for this change in your PR description. Please link the issue. This is not required.